aws/neumann_buildspec.yml

```
@@ -0,0 +1,49 @@
+version: 0.2
+
+env:
+  secrets-manager:
+    DOCKERHUB_PASS: "dockerhub/credentials:password"
+    DOCKERHUB_USERNAME: "dockerhub/credentials:username"
+
+phases:
+  pre_build:
+    commands:
+      - echo Build started on `date`
+  build:
+    commands:
+      - echo Logging in to Docker hub...
+      - docker login --username $DOCKERHUB_USERNAME --password $DOCKERHUB_PASS
+      - IMAGE_TAG=$(git describe --abbrev=0)
```

This will output a tag with the format `v1.2.7.1`.
If you want to cut off the leading `v` you can use cut
```suggestion
      - IMAGE_TAG=$(git describe --abbrev=0 | cut -c 2-)
```